locale/de_DE/LC_MESSAGES/django.po

```diff
@@ -5486,6 +5497,34 @@ msgstr ""
 "Sie sind dabei sich mit Ihrem %(provider_name)s-Konto auf %(site_name)s "
 "anzumelden. Füllen Sie bitte das folgende Formular als letzten Schritt aus:"
 
+#~ msgid "Moderator feedback"
+#~ msgstr "Rückmeldung der Moderation"
+
```

Comment:
Fine for now, but next time, you can delete the old translations. Here and in the English files.